tyr-core/src/main/java/org/jboss/tyr/config/TyrConfiguration.java

```diff
@@ -23,6 +23,9 @@
     @ConfigProperty(name = "github.status.push", defaultValue = "true")
     boolean statusPush();
 
+    @ConfigProperty(name = "github.commit.check.disabled", defaultValue = "false")
```

Comment:
```suggestion
    @ConfigProperty(name = "github.commit.checks.disabled", defaultValue = "false")
```
There might be more commit checks in future (almost for sure will be :D)

Reply:
Implemented fix based on recommendation.